lib/qops/deployment/instances.rb

```diff
@@ -152,7 +152,7 @@ def clean
 
     # Find all instances to be destroyed
     retrieve_instances.each do |instance|
-      next if instance.hostname == 'master'
+      next if instance.hostname == "#{config.hostname_prefix}master"
```

Comment:
Does the regex really make sense? Is it possible the word "master" might show up in a valid branch name? I think my solution keeps us closer to the intended spirit, which is JUST the master branch, just bringing into account the already-configured prefix value.

I agree with tagging though - definitely more elegant than hostnames.